WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1988,9 +1988,8 @@ private void removeNonSelfHostedPreferences() {
      */
     private void removeGeneralSettingsExceptBloggingReminders() {
         PreferenceGroup group = (PreferenceGroup) findPreference(getString(R.string.pref_key_site_general));
-        if (group != null && mBloggingRemindersPref != null) {
+        if (group != null) {
             group.removeAll();
-            group.addPreference(mBloggingRemindersPref);
         }
     }
```

Comment:
After this change, `removeGeneralSettingsExceptBloggingReminders()` removes everything. I think it would be better to rename the function.